### ISSUE TITLE
PIA PF endpoints' SSL certs are expiring/have expired

### ISSUE THREAD

**Issue by TheWug** [pia-foss/manual-connections#93]
I don't know if this repo is a completely unaffiliated project or if PIA staff have a hand in it; therefore I'm not sure if this is the correct place to raise these issues, but several PIA endpoints are using SSL certificates which have now expired.

You can see this easily for yourself by attempting to view the following in browser:
https://ca-toronto.privacy.network:19999 (expired for 87 days)
https://ca-ontario.privacy.network:19999 (expired for 4 days)
https://ca-montreal.privacy.network:19999 (expired for 4 days)
https://ca-vancouver.privacy.network:19999 (expired since yesterday)

I only checked the canadian servers, but it's likely that the majority of the fleet's certs are expired.

Can someone look into updating them, and/or raising the issue with PIA so they can look into that?

**Comment by g00nix** [comment]
TL/DR:

use `curl -k`.

---

> I don't know if this repo is a completely unaffiliated project or if PIA staff have a hand in it; therefore I'm not sure if this is the correct place to raise these issues, but several PIA endpoints are using SSL certificates which have now expired.

This is an official repository and this issue section should be dedicated to issues regarding the source code of this repository.

> You can see this easily for yourself by attempting to view the following in browser:

The PF API was not designed to be used in a browser. For proper usage, please refer to [the PF script](https://github.com/pia-foss/manual-connections/blob/master/port_forwarding.sh).

> Can someone look into updating them, and/or raising the issue with PIA so they can look into that?

You should use the self signed certificate, as described in this repo. I will however restart the PF APIs on all nodes so that they all get the new certificate, though this is not really supported by current scripts.

Closing as a duplicate of https://github.com/pia-foss/manual-connections/issues/85 and https://github.com/pia-foss/manual-connections/issues/92
